action.yml

```diff
@@ -47,7 +47,24 @@ inputs:
     description: 'a boolean to support dry run'
     required: false
     # Application Default - false
-
+  jira-link-id:
+    description: 'the new created issue will be linked to this link id'
+    required: false
+  jira-link-type:
+    description: "this defines the relation of the new issue with feature issue specified and default value for this is 'Relates'"
+    required: false
+  jira-link-direction:
+    description: "this variable is to specify the direction of linked issue either inward or outward"
+    required: false
+  include-all-products:
```

Comment:
include-all-products and skip-products seem potentially confusing, why would you include all products to then skip products?

Maybe an optional setting 'include-projects' that would default to a value of a list only containing ["Security Hub"], but you could override it to provide a list of whatever products you wanted?

Reply:
All products is to allow issue creation for all products. Instead of just SecHub product

Skip product is an additional feature that will add all products EXCEPT those specified. (This use will be incase one or two products are not wanted) This is to allow selection if there is some product types that they don’t want issues for. If we don’t have this unwanted issues will be created. 

If we don’t have skip product they will only have two options: all or sechub. 

This covers the mention by tmsis- what if we don’t want all products. Like product;  “health”

Reply:
Using an include-products variable is impractical because:

Long Lists: You'd have to specify every product to include, which is cumbersome, especially with many products (e.g., JFrog, NASH, Default, Inspector, GuardDuty, etc Tmsis has 6-7 products atm and list will be growing).
Frequent Changes: Adding new products would require updating the workflow file every time.
In contrast, using a skip-products variable is more efficient because:

Shorter Lists: You only list the products to exclude, which is typically shorter and easier to manage.
Infrequent Changes: The workflow file changes less often since the skip list is updated only when exclusions are needed. Which will most likely only be Default and health products.

Reply:
Its understandable if those are the requirements, however still think there are some potential issues with this plan.

For example if say you only care about 2 products and you don't want new products to start creating tickets without you setting it up first (I would find it insanely annoying to have some new product show up with 40 false positives out of nowhere and now I have a pile of tickets to close) then you don't really have a choice, you have to include all products and then exclude the ones you don't care about. However then when a new product is released its going to just start creating tickets for it until you add that to the exclude list.

I'd probably lean more towards something like:
include-all-products: defaults to false
include-products: defaults to ["Security Hub"]
exclude-products: defaults to [""]

An adjustment to this could be to support a wildcard product "*" in include-products and then we could drop include-all-products.

I'm sure there are a lot of other ways to handle all of these cases if you have other ideas.

Reply:
I understand your point here, Ill take this up and add this on our backlog for review -as additional teams onboard to V2 we can escalate this if it is an item that is challenged.